demographics.html

```diff
@@ -12,22 +12,24 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
     <title>Austin Code's GitHub</title>
-    <!-- Bootstrap core CSS -->
-    <link href="css/bootstrap.min.css" rel="stylesheet">
-    <!-- Bootstrap theme -->
-    <link href="css/bootstrap-theme.min.css" rel="stylesheet">
+    <!-- Latest compiled and minified CSS -->
```

Comment:
i'm confused as to why this update is overwriting old text?

there should be no change in this section except for the css/navbarcustom.css